### ISSUE TITLE
Code snippet not changed over in Set Up Rust

### ISSUE THREAD

**Issue by kylegalloway** [phil-opp/blog_os#322]
In the https://github.com/phil-opp/blog_os/blob/set_up_rust/Makefile, the section for compiling with cargo is not updated to compiling with xargo.

It should be:
```
$(kernel): kernel $(rust_os) $(assembly_object_files) $(linker_script)
	@ld -n --gc-sections -T $(linker_script) -o $(kernel) \
		$(assembly_object_files) $(rust_os)

kernel:
	@xargo build --target $(target)
```
This also needs to be updated in the book at http://os.phil-opp.com/set-up-rust.html#gc-sections.

**Comment by phil-opp** [comment]
Thanks for reporting! Sorry for the late reply, I somehow missed this…

We deprecated the `set_up_rust` tag in #320 in favor of the new `post_3` tag (which [uses xargo](https://github.com/phil-opp/blog_os/blob/post_3/Makefile)). I thought we removed all links to the `set_up_rust` tag, but maybe we missed some…

> This also needs to be updated in the book

Good catch! Fixed in 160b98632279f9cd4aa328fdec15b66b9b3c4d72.

**Comment by phil-opp** [comment]
I just deleted the old tags (e.g. `set_up_rust`) in favor of the new `post_X` tags. Please open issues if you find dead links or have any other problems with this deletion.
